Formula/akamai.rb

```diff
@@ -12,25 +12,22 @@ class Akamai < Formula
     sha256 cellar: :any_skip_relocation, mojave:   "1d96850b0c979f5f351877977b7d06e5b26a78701a8993ad6366b229c15cae97"
   end
 
-  depends_on "dep" => :build
   depends_on "go" => :build
 
   def install
     ENV["GOPATH"] = buildpath
     ENV["GO111MODULE"] = "auto"
-    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
 
     srcpath = buildpath/"src/github.com/akamai/cli"
     srcpath.install buildpath.children
 
     cd srcpath do
-      system "dep", "ensure", "-vendor-only"
-      system "go", "build", "-tags", "noautoupgrade nofirstrun", "-o", bin/"akamai"
+      system "go", "build", "-tags", "noautoupgrade nofirstrun", "-o", bin/"akamai", "cli/main.go"
       prefix.install_metafiles
     end
```

Comment:
Since you use Go modules now, I think we can get rid of this entire block and replace it with just
```ruby
system "go", "build", "-tags", "noautoupgrade nofirstrun", *std_go_args, "cli/main.go"
```

Reply:
great suggestion, thanks!